.github/workflows/nightly-gauntlet.yaml

```diff
@@ -3,22 +3,27 @@
 name: nightly-gauntlet
 on:
   schedule:
-    # Every day at midnight
-    - cron: "0 0 * * *"
+    # Every day at 4AM
+    - cron: "0 4 * * *"
```

Comment:
It is a good idea considering all flakes around insights and day-change/midnight-related issues.

Reply:
Could we modify cron not to alarm during weekends? Nobody is expected to take an action until Monday morning?

Reply:
OK, will use @matifali's suggestion below.